recipe/build.sh

```diff
@@ -42,14 +42,16 @@ function bootstrap_build {
 # https://www.erlang.org/doc/installation_guide/install-cross#Build-and-Install-Procedure_Building-With-configuremake-Directly_Building-a-Bootstrap-System
 if [[ "${CONDA_BUILD_CROSS_COMPILATION}" -eq 1 ]]; then
   bootstrap_build
+  # erl_xcomp_sysroot is needed for cross-compilation to find SSH headers for target arch.
+  # https://github.com/erlang/otp/blob/master/HOWTO/INSTALL-CROSS.md#cross-system-root-locations
+  export erl_xcomp_sysroot="${CONDA_BUILD_SYSROOT}"
 fi
 
 ./configure \
     --prefix="${PREFIX}" \
     --with-ssl="${PREFIX}" \
```

Comment:
I am a little confused here. `$PREFIX` should point to directories where erlang will be installed, while `--with-ssl` should point to a directory, where to look for SSL header files and such. Shouldn't they be different? And specifically for cross-compilation, I'd assume that it should be `--with-ssl=${CONDA_BUILD_SYSROOT}`, so that we're compiling against the correct headers.

Reply:
Nope using `$PREFIX` is right here as that is where all `requirements/host` dependencies will be installed

`$CONDA_BUILD_SYSROOT` is used for system dependencies. So `cdt`s for graphics drivers, etc..